common/production_methods/07_government.txt

```diff
@@ -407,7 +407,128 @@ pm_religious_academia = {
 		}
 	}			
 }			
+pm_no_academic_focus = {
+	texture = "gfx/interface/icons/production_method_icons/unused/no_philosophy_dept.dds"
+}
+pm_military_academy = {
+	texture = "gfx/interface/icons/production_method_icons/unused/military_academy.dds"
+	building_modifiers = {
+		level_scaled = {
+			building_employment_officers_add = 150
+            building_employment_academics_add = 50
+            building_employment_soldiers_add = 250
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 0.5
+			building_input_small_arms_add = 1
+		}	
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_military_tech_spread_mult = 0.01
+		}
+	}
+}
+
+pm_research_institute = {
+	texture = "gfx/interface/icons/production_method_icons/unused/engineering_university.dds"
+	building_modifiers = {
+		level_scaled = {
+			building_employment_academics_add = 100
+            building_employment_engineers_add = 100
+            building_employment_machinists_add = 150
+            building_employment_laborers_add = 100
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 1
+			building_input_services_add = 0.5
+		}	
+	}
+	country_modifiers = {
+		workforce_scaled = {
+			country_production_tech_spread_mult = 0.01
+		}
+	}
+}
+
+pm_liberal_arts = {
+	texture = "gfx/interface/icons/production_method_icons/unused/social_science.dds"
+	building_modifiers = {
+		level_scaled = {
+			building_employment_academics_add = 150
+            building_employment_bureaucrats_add = 50
+            building_employment_clerks_add = 100
+            building_employment_laborers_add = 100
+		}
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			building_input_paper_add = 0.5
+			building_input_services_add = 1
```

Comment:
I assume you already calculated that all the pms get to the same goods price when considering the base price right?

So paper + services in this pm is the same as paper + small arms in another.

Reply:
I did not in the case of the Military, I think them costing $30 more than the others is appropriate, and not too outrageous an increase. But if you feel like they should be the same cost throughout I'm ok with that.

Also, for general goods since it is the most potentially powerful one, I let that be more expensive than the specified counterparts.

Reply:
Well it's 40% more so do with that what you want.